_includes/footer.html

```diff
@@ -7,7 +7,7 @@
 				<div class="row">
 					&copy; 2017 Developer Club. All rights reserved.
 				</div>
-				<div class="row">
+				<div class="row my-auto">
```

Comment:
A feature of Bootstrap being used here, always remember that Bootstrap is available to be used.